docs/core/toc.yml

```diff
@@ -81,59 +81,95 @@
   href: compatibility/
 - name: Tutorials
   items:
-  - name: Overview
-    href: tutorials/index.md
-  - name: Get started with .NET Core using Visual Studio
-    href: tutorials/with-visual-studio.md
-  - name: Get started with .NET Core using Visual Studio Code
-    href: tutorials/with-visual-studio-code.md
-  - name: Get started with .NET Core using the CLI
-    href: tutorials/cli-create-console-app.md
-  - name: Get started with .NET Core using Visual Studio Code on macOS
-    href: tutorials/using-on-macos.md
-  - name: Get started with .NET Core using Visual Studio for Mac
-    href: tutorials/using-on-mac-vs.md
-  - name: Debug an application using Visual Studio
-    href: tutorials/debugging-with-visual-studio.md
-  - name: Publish an application with Visual Studio
-    href: tutorials/publishing-with-visual-studio.md
-  - name: Create a .NET Standard library in Visual Studio
-    href: tutorials/library-with-visual-studio.md
-  - name: Test a .NET Standard library in Visual Studio
-    href: tutorials/testing-library-with-visual-studio.md
-  - name: Consume a .NET Standard library in Visual Studio
-    href: tutorials/consuming-library-with-visual-studio.md
-  - name: Build a complete .NET Core solution on macOS
-    href: tutorials/using-on-mac-vs-full-solution.md
-  - name: Organize and test projects with the .NET Core CLI
-    href: tutorials/testing-with-cli.md
-  - name: Develop libraries with cross-platform tools
-    href: tutorials/libraries.md
-  - name: Create a .NET Core app with plugins
-    href: tutorials/creating-app-with-plugin-support.md
-  - name: Develop ASP.NET Core apps
-    href: tutorials/aspnet-core.md
```

Comment:
@tdykstra Was this tutorial excluded from the updated TOC on purpose?

Reply:
plugins moved to the Advanced section. ASP.NET Core is included in "More tutorials"

```
  - name: More tutorials
    href: tutorials/index.md
```

Reply:
I was referring to aspnet-core.md, sorry. I don't see it in https://raw.githubusercontent.com/dotnet/docs/master/docs/core/tutorials/index.md.

Reply:
ASP.NET Core is covered by these lines

```
## Create web apps

For tutorials about developing ASP.NET Core web applications, see the [ASP.NET Core documentation](/aspnet/core/).
```

aspnet-core.md should have been deleted.